tests/integration/render.rs

```diff
@@ -1,52 +1,139 @@
 use assert_cmd::prelude::*;
+use cmd_lib::run_fun;
+use std::fs::OpenOptions;
 use std::io::Read;
+use std::io::{self, Write};
 use std::path::PathBuf;
 use std::process::Command;
 
 const BIN_NAME: &'static str = env!("CARGO_PKG_NAME");
 
+fn generate_config() -> Result<PathBuf, io::Error> {
+    // Generate the Config in string literal
+    let config = r#"---
+version: v2
+enabled: true
+chart: ../nginx-chart
+namespace: my-namespace
+release_name: my-app
+output_path: manifests
+additional_options:
+  - "--skip-crds"
+  - "--no-hooks"
+values:
+  - ../nginx-chart/values/default.yaml
+deployments:
+  - name: edge-eu-w4
+    values:
+      - ../nginx-chart/values/edge.yaml
+    additional_options:
+      - "--set image.tag=latest"
+  - name: next-edge-eu-w4
+    enabled: false
+    values:
+      - ../nginx-chart/values/edge.yaml
+      - ../nginx-chart/values/next-edge.yaml
+  - name: stage-eu-w4
+    values:
+      - ../nginx-chart/values/stage.yaml
+  - name: prod-eu-w4
+    release_name: my-app-prod-eu-w4
+    values:
+      - ../nginx-chart/values/prod.yaml
+      - ../nginx-chart/values/prod-eu-w4.yaml
+"#;
+
+    // Create a temp folder
+    let config_folder = run_fun!(mktemp -d "tests/data"/configXXXX)?;
+
+    let path = format!("{}/config.yaml", config_folder);
+
+    // Create a config file and put it in the config folder
+    let mut tmp_file = OpenOptions::new()
+        .write(true)
+        .read(true)
+        .create(true)
+        .open(&path)
+        .unwrap();
+
+    // Write the generated string literal to a file in the temp folder
+    writeln!(tmp_file, "{}", config)?;
+
+    // return the entire folder
+    Ok(PathBuf::from(path))
+}
+
+fn drop_temp_folders(dir: &str) -> Result<String, io::Error> {
+    // delete the tmp folder
+    Ok(run_fun!(rm -r $dir).unwrap())
+}
+
 #[test]
 fn render_config_example() -> anyhow::Result<()> {
     let mut cmd = Command::cargo_bin(BIN_NAME)?;
 
-    cmd.current_dir("tests/data")
-        .arg("render")
-        .arg("config_example.yaml");
+    let config = generate_config().unwrap();
```

Comment:
```suggestion
    let config = generate_config()?;
```
Since the test returns a Result type.